apis/v1alpha1/httproute_types.go

```diff
@@ -312,7 +313,11 @@ type RouteHostExtensionObjectReference = ConfigMapsDefaultLocalObjectReference
 
 // HTTPRouteStatus defines the observed state of HTTPRoute.
 type HTTPRouteStatus struct {
-	GatewayRefs []GatewayObjectReference `json:"gatewayRefs" protobuf:"bytes,1,rep,name=gatewayRefs"`
+	// Gateways is a list of the Gateways that are associated with the
+	// HTTPRoute, and the status of the route with respect to each of these
+	// Gateways.  Each associated Gateway should have exactly one entry in
+	// this list.
```

Comment:
We should add some kind of clarification as to what controllers should or must do here, assuming controllers are responsible for setting this status.

Reply:
Basically the controller should set the status condition to either `True` or `False` when it sees the route, and possibly update that status condition when the route is modified.  I can add a statement to that effect.  I did have a question about cleaning up old status entries if a controller that has added a status condition is subsequently deleted (https://github.com/kubernetes-sigs/service-apis/pull/86#discussion_r379168521); this is still an open question that I'm not sure how best to address.